modules/richards/tests/gravity_head_2/tests

```diff
@@ -171,4 +171,43 @@
     rel_err = 1e-5
     abs_zero = 1e-4
   [../]
+
+  [./except01]
+    type = 'RunException'
+    input = 'gh01.i'
+    cli_args = 'Materials/rock/viscosity="1"'
+    expect_err = 'There are 2 Richards fluid variables, so you need to specify this number of viscosities, relperm_UO, seff_UO, sat_UO, density_UO, SUPG_UO'
+  [../]
+  [./except02]
+    type = 'RunException'
+    input = 'gh01.i'
```

Comment:
I don't think you can remove a root level block with `cli_args`. I was thinking more along the lines of you removing it from the input file itself. Most of these checks don't need it. You could add it instead on the one case that needs it, or create a separate input file altogether for a RunException checks.